main.go

```diff
@@ -154,12 +161,48 @@ func setupChecks(mgr ctrl.Manager) {
 	}
 }
 
+// setupReconcilers can either create a client for an in-cluster installation (rancher and rancher-turtles in the same cluster)
+// or create a client for an out-of-cluster installation (rancher and rancher-turtles in different clusters).
 func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
+	var c client.Client
+
+	if len(rancherKubeconfig) > 0 {
+		setupLog.Info("out-of-cluster installation of rancher-turtles", "using kubeconfig from path", rancherKubeconfig)
+
+		restConfig, err := loadConfigWithContext("", &clientcmd.ClientConfigLoadingRules{ExplicitPath: rancherKubeconfig}, "")
+		if err != nil {
+			setupLog.Error(err, "unable to load kubeconfig from file")
+			os.Exit(1)
+		}
+
+		c, err = client.New(restConfig, client.Options{Scheme: mgr.GetClient().Scheme()})
+
+		if err != nil {
+			setupLog.Error(err, "failed to create client")
+			os.Exit(1)
+		}
+	} else {
+		setupLog.Info("in-cluster installation of rancher-turtles")
+		c = mgr.GetClient()
+	}
+
 	if err := (&controllers.CAPIImportReconciler{
-		Client:           mgr.GetClient(),
+		Client:           c,
```

Comment:
We actually need 2 clients for `CAPIImportReconciler`:

- 1 for the controller itself for the CAPI Clusters
- 1 for Rancher

So maybe something like this:

```go
type CAPIImportReconciler struct {
	Client client.Client
        RancherClient client.Client
```

In the scenario where Rancher Manager and CAPI are in the same cluster then:
- **Client** is set to `mgr.GetClient()`
- **RancherClient**  is set to `mgr.GetClient()`

In the scenario for this change (i.e. where the CAPI management cluster is different to the Rancher Manager cluster) , then:
- **Client** is set to `mgr.GetClient()`
- **RancherClient** is set to client created from the kubeconfig specified via **rancherKubeconfig**

How does that sound?

Reply:
@richardcase updated the PR with your suggestions.